spi/build.gradle

```diff
@@ -7,15 +7,15 @@ import com.github.jengelman.gradle.plugins.shadow.ShadowBasePlugin
 import org.opensearch.gradle.test.RestIntegTestTask
 
 plugins {
-    id 'com.github.johnrengelman.shadow'
+    id 'java'
+    id "io.freefair.lombok"
```

Comment:
seems that lombok is not used in spi sub project

Reply:
Thanks for catching this. Done.